R/datasets.R

```diff
@@ -362,3 +362,20 @@
 #' }
 #' @seealso This data was sampled from the `movies` data frame in the `ggplot2movies` package.
 "movies_sample"
+
```

Comment:
You have two `.rda` files in the `data` folder. Please delete the one that is not nessecary.